### PR TITLE
Update: usb-modeswitch 2.2.0 -> 2.2.1

### PR DESCRIPTION
**Pull request by ikervagyok**

### CODE REVIEW

File: pkgs/development/tools/misc/usb-modeswitch/default.nix [hunk 1]
```diff
@@ -1,11 +1,11 @@
 { stdenv, fetchurl, pkgconfig, libusb1 }:
 
 stdenv.mkDerivation rec {
-  name = "usb-modeswitch-2.2.0";
-
+  name = "usb-modeswitch";
```
Comment (nckx) [inline pkgs/development/tools/misc/usb-modeswitch/default.nix]: Why not `name = "usb-modeswitch-${version}"`?

**Comment by jagajaga** [comment]
Please, change as @nckx suggested :)

**Comment by ikervagyok** [comment]
@jagajaga @nckx: changed... also: what is the preferred way to do name/version stuff? is there a standard, which the project follows?

**Comment by nckx** [comment]
Sadly, no. My understanding of `name` is that it's a relic from the time when attribute names weren't descriptive enough to map to common package names. The main problem was that dashes weren't allowed.

That's no longer the case. I've heard talk - no idea where or by whom - of getting rid of `name` entirely, presumably in favour of an unambiguous `version` attribute. I think that's the right way to go.

Meanwhile, this method, my preferred `let version = "2"; in`, or just `name = "foo-2"` are all accepted.

**Comment by jagajaga** [comment]
Now please squash commits into one.

**Comment by ikervagyok** [comment]
is it ok now? i changed to "let version =...; in", as it seems to be the best way of @nckx 's ways to do it

**Comment by jagajaga** [comment]
It's not the best or the worst way, it's just a way :)

**Pull request merged**